Formula/git-flow.rb

```diff
@@ -44,11 +44,15 @@ def install
     bin.write_exec_script libexec/"bin/git-flow"
     resource("completion").stage do
       bash_completion.install "git-flow-completion.bash"
-      zsh_completion.install "git-flow-completion.zsh" => "_git-flow"
-      inreplace "#{zsh_completion}/_git-flow", "#!zsh", "#compdef git-flow"
     end
   end
 
+  def caveats
+    <<~EOS
+      Completions for ZSH are included in `brew install zsh-completions`
```

Comment:
```suggestion
      To install Zsh completions:
        brew install zsh-completions
```
Let's make this a bit more consistent with other formulae.